docs/applications/github-action/03-azure.md

```diff
@@ -0,0 +1,132 @@
+---
+id: applications-github-action-azure
+slug: /applications/github-action/azure
+title: Azure
+sidebar_label: Azure
+---
+
+## Docker image push & deploy
+The Massdriver CLI supports pushing images to an Azure Container Registry (ACR) and deploying them to Massdriver. This guide will walk you through the process of setting up a GitHub Action to push app changes to your ACR repository and deploy them to Massdriver.
+
+:::note
+
+Before getting started, you'll need:
+- A Massdriver organization 
+- A Massdriver [service account](/platform/service-accounts)
+- The [Massdriver CLI](https://github.com/massdriver-cloud/massdriver-cli) installed
+
+:::
+
+### Publish your application
+Before you can set up a GitHub Action to deploy your application, first you'll need to publish it to Massdriver and create a package. You can do this by following the [Getting Started](/applications/getting-started) guide.
+
+After you publish your application, you'll need to create a package. You can do this by dragging your application out from the bundle bar in the Massdriver canvas. Fill in the values of your application and click **Deploy**.
+
+Copy and save the package slug by hovering over the name of your application on the canvas. This will be used in the next step.
+
+### Package preview file
+Next, we'll need to create a package preview file to fetch the application params. This file will be used by the GitHub Action workflow to update the image tag and update the configuration of your application.
+
+To generate this file, in your terminal run the following command:
+
+```bash
+mass preview init <application-package-slug-id>
+```
+
+Hover over `Azure` and hit **enter** to select. Hit `s` to save. Hover over the name of your credential and hit **enter** to select. Hit `s` to save.
+
+This will output a JSON file named `preview.json` in your current directory with the values of every package in the same target as your application. Use `cat preview.json` to view the contents of the file. Copy and save the `credentials` UUID for use in [Set secrets and vars](#set-secrets-and-vars).
+
+Find the name of your application and copy the JSON block for your application into a new file named `mass-app-params.json`. You can find an example params file below:
```

Comment:
I don’t understand why we’re doing this. This block is for controlling your preview environments only and the preview command understands what to do with it. We shouldn’t need to copy it to another file.